packages/react/src/components/ResourceList/ResourceList.jsx

```diff
@@ -192,16 +154,17 @@ class ResourceList extends Component {
     });
 
     return (
-      <ResourceListSection
+      <section
         data-testid={testId}
         className={[
+          `${iotPrefix}--structured-list`,
```

Comment:
Perhaps call this `${iotPrefix}--resource-list` ?

Reply:
Fixed